lib/project_types/script/layers/infrastructure/assemblyscript_dependency_manager.rb

```diff
@@ -41,15 +41,13 @@ def write_package_json
               "version": "1.0.0",
               "devDependencies": {
                 "@shopify/scripts-sdk-as": "#{@extension_point.sdks[:ts].sdk_version}",
+                "@shopify/scripts-toolchain-as": "#{@extension_point.sdks[:ts].toolchain_version}",
                 "#{@extension_point.sdks[:ts].package}": "#{@extension_point.sdks[:ts].version}",
                 "@as-pect/assembly": "3.1.1",
                 "@as-pect/cli": "3.1.1",
                 "@as-pect/core": "3.1.1",
-                "yargs": "10.0.0",
```

Comment:
I removed this dependency as it should be a depenendency of the EP (since it's only used for the bootstrap script).